packages/@coorpacademy-components/src/molecule/learning-priority-modal/index.js

```diff
@@ -50,11 +50,6 @@ const FilterButton = (props, context) => {
   return <ButtonLink {...buttonProps} />;
 };
 
-FilterButton.contextTypes = {
-  skin: Provider.childContextTypes.skin,
-  translate: Provider.childContextTypes.translate
```

Comment:
Sure the filterButton doesnt need the translate?

Reply:
It doesn't as the label is translated in coorpmanager before passed in 👍